V6_MIGRATION.md

```diff
@@ -164,3 +166,17 @@ sepaDirectDebitClient.tokenize(request: sepaDirectDebitRequest) { sepaDirectDebi
     // handle response
 }
 ```
+
+## Local Payment Methods
```

Comment:
It looks like these instructions are the same for both headers. Maybe we can instead consolidate them

Reply:
Yeah we went back and fourth on this. At first we used a "Payment Flow" header but weren't sure if we needed to be more explicit about calling out LPMs and 3DS specifically. I guess we could also use the "Payment Flow" header and under it callout that it applies to 3DS and LPMs before going into the changes. Thoughts?

Reply:
Gotcha - I personally prefer the later option. Or maybe even the header could be `## Local Payments & 3DS`?

Another thing - shall we add details that they can remove the [custom URL scheme](https://developer.paypal.com/braintree/docs/guides/client-sdk/setup/ios/v5#register-a-url-type) they previously added for BT?

Reply:
Good call on the URL scheme - forgot about it because it's still used for Venmo so it still exists in the demo. Updated with the above changes here: https://github.com/braintree/braintree_ios/pull/952/commits/c78e3af922d931f8a56e5cce9a057f77b11eb9ad